.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - main
+      - "release-*"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+env:
+  MAIN_BRANCH_NAME: main
+
+jobs:
+  build:
```

Comment:
`build` doesn't seem like a name that describes what this job is doing. It could be `test`.